base/multidimensional.jl

```diff
@@ -1298,9 +1298,16 @@ hash(x::Prehashed) = x.hash
 
 Returns an array containing only the unique elements of the iterable `itr`, in
 the order that the first of each set of equivalent elements originally appears.
+The unicity is determined by [`isequal`](@ref).
```

Comment:
I think "unicity" is a fairly uncommon word. (Indeed, Firefox spellcheck is marking it as incorrect, though a Google search shows it exists.) I think "uniqueness" might be clearer here.